### ISSUE TITLE
2.2.1.1. "Question" as a Prompt

### ISSUE THREAD

**Issue by ARMediting** [gryphonshafer/Quizzing-Rule-Book#133]
I'd like to propose shortening "What is your question?" as a QM prompt to simply "Question." Anything we can do to interfere less with the quizzer's answering should be considered. It is also more accurate, as "your question" suggests that whatever a quizzer provides as "their question" will be accepted.

Also, in conjunction with my suggestion in Issue #28 to remove "More" as a prompt, I propose using "Question" as the prompt for finishing a verse number on Quotes. And in Reference questions, "Question" should serve as the prompt for providing the end of a number if needed as well as the question itself.

As to whether this should be used with Situations, where the quote technically is part of the question but doesn't make a lot of sense that way, I think "Again" and "Quote is complete" should suffice. 2.2.1.1.5. already includes "More" as a prompt for when the quotation isn't complete, but that seems redundant with "Quote is complete."

**Comment by jttower** [comment]
I like the idea of streamlining prompts whenever possible and I think using "Question" as the same prompt in any case where we need more info from the question makes sense.  I think this would work well for INT, MA, REF, and Q.  FTV shouldn't need any prompts other than "Again". 

For SIT, I agree that we should stick with "Again" if they have gotten to the end but missed something (just like we would for Q/F), and I think "Quote is complete" works well when they have completed the quote.

**Comment by jswingle** [comment]
Changing it to be a universal prompt for completing information found in the question -- 100% yes. I like the consistency of the proposal - if Quotes it's just the reference itself, but on a Reference it could be the verse as well as the question. 

I agree that with SITs we should leave it at "Quote is complete" because this question type is categorically different. In a SIT we ask the questions _before_ the jumping period begins, they're always fully given. FTVs are also categorically different, but as has already been mentioned, that's covered by "Again."

The only downside I can see to shortening the prompt is that it requires a bit more rulebook knowledge for a newer quizzer to make sense of things, but for the most part quizzing is like that anyways - there's already a "jargon learning curve." The value add to making the prompt shorter and sweeter far outweighs the downside.

**Comment by ZacharyTinker** [comment]
Leave a comment on the rule change proposal (GitHub Issue)

**Comment by josiah-leinbach** [comment]
To channel my inner baseball fan, I appreciate "the human element" of quizzing and the reasonable variety between quizmasters, as that forces quizzers to adapt to different styles. Practice-driven logic leads me to think that, in the real world, very few errors happen as a result of an ambiguous or misleading prompt of some sort because (1) ambiguous or misleading prompts are themselves rare; (2) a quizzer who has given sufficient information to warrant a prompt either knows the material well such that the content of the prompt need be only vaguely relevant to point the quizzer in the right direction, or the quizzer doesn't know the material well enough and not even the clearest prompt would help them; and (3) we add a new layer of complexity not just for quizmasters but also for quizzers. In my opinion, if a quizzer can successfully challenge a ruling on the basis of technically invalid but intuitively obvious prompt, we have ventured into the realm of the pedantic.

In short, I do not think the rulebook should prescribe any quizmaster prompts with specific wording.

**Comment by scottpeterson** [comment]
I am heartily in favor of 3 things:
* Quizmaster prompts that are as short as possible (so in support of Alex's initial proposal)
* Quizmaster prompts that are as limited as possible (e.g. I don't think the "Again" or "More" prompts are necessary)
* Clear, objective definitions that require the Quizmaster to say prompts

Quizmasters exist to facilitate the competition, not to be a part of it. Ideally QM prompts are 100% consistent, always interpreted the same way by quizzers, effectively letting QMs fade into the background. 

I am against any "human element" of quizmastering that leads to different levels of information presented to different quizzers. 


My only caution with Alex's proposal is there may be a short "getting used to the change" period, since "What is your question?" is a clear "asking phrase", but the word "question" is not.   QMs could be clear by pronouncing "question" with an up-inflection.

**Comment by jswingle** [comment]
This is also one of those issues where, even though I think this is clearly the right approach, a lot of people will be understandably taken aback by it. This is a pretty big change, in the sense that it's very obvious and quizzer/coach-facing.

Important to consider as we're talking about this. It could be a good issue to bring up at Internationals this year when we present on the rulebook project and how people can collaborate on it.

**Comment by jswingle** [comment]
I'd like to start making progress on this issue because I think a full-scale revision of prompts is a reasonable goal for next year's rulebook with a significant value add for quizzing. We have numerous requests for fixing prompt-related rules, so we should try to roll all those issues into one rulebook revision.

This is one of the simpler issues related to prompts. The language change proposal:
2.2.1.1.2 and 2.2.1.2 both specify the prompt "What is your question?", and we would just change both of these to "Question?"

@kclimenhaga has expressed the difference as a matter of politeness. "What is your question?" would certainly be more polite in everyday speech, but when it comes to prompts, I think we just need to pick something that is clear and works. I think of it more like "Check" or "Checkmate" in chess - the point is not to express politeness or triumph or any other emotion, it's just a part of the rules that you have to say it in order for the game to function properly. Same with QM prompts.

As @scottpeterson alluded to above, we sort of want the QM to fade into the background. We're just there to give the prompt at the right time, and I think making the prompts as "prompt" as possible gives the quizzer more of their 30 seconds to do their thing.

So in this case, I would be in favor of shortening the prompt to "Question?" I don't think we should give two options to the QM -- the prompt should be standardized and always the same. (Likewise, the "can you identify _pronoun_" prompt should be shortened to just "identify _pronoun_" or, even better, "clarify _pronoun_", but this belongs in a separate issue)

**Comment by levikoral** [comment]
I like the idea of shortening it to "Question."

**Comment by JoshJetto** [comment]
There are probably two parts of this section of the rulebook that I think really need fixing. The first is to change the prompt when the reference is needed from "More" to "What is your reference?" or "Your reference?" because "more" in this instance is very, very confusing. The second part would be to append the commentary section with the following: "If the quizmaster has given a prompt that is clear and is in keeping with the spirit of these rules concerning what prompts may be used, even if not verbatim per these rules, such is acceptable - provided they have not given a prompt at a time when it is impermissible to do so." This allows for the human element in quizmastering and gives the quizmasters latitude to give more time when time is short by using a shorter prompt and to give more clarity when time is not short. I think the point of the rules on prompts should not be to create more minutiae to serve as fodder for challenges; it should be to provide the quizzers with clarity and fairness as well as to provide the average spectator with an understandable experience of quizzing.

**Comment by josiah-leinbach** [comment]
@JoshJetto Do you think that both changes (i.e. the change to "more" and this proposed change) would need to be enacted simultaneously, or can they be enacted separately?

**Comment by JoshJetto** [comment]
I think either way is fine. I think both changes would be helpful, but if we want to vote separately on each one, we can. I'd ask the RC when we get to this agenda item.

**Comment by jttower** [comment]
See Issue #175 for an update on this issue.

**Comment by jttower** [comment]
As was posted on Issue #175 , at the Rules Committee meeting on 4/2/23, we voted 5-1 to change to the following:
—-----------------------------------
3.2.1.1. Quizmaster Prompts
During the answering period, the quizmaster will prompt the quizzer in the following ways. If the quizmaster has given a prompt that is clear and is in keeping with the spirit of these rules even if not word for word, it is acceptable as long as the occasion for the prompt is still correct.
3.2.1.1.1. Clarify
If a quizzer has provided all the information in the question and answer, except the identification of a pronoun present which is part of the answer text, the quizmaster will prompt the quizzer to identify the specific pronoun. For example: "Can you clarify 'He'?" or "Clarify 'They.'"
The quizmaster is not allowed to say anything other than what the quizzer has already said when asking for the identification. A pronoun may need to be identified only if the antecedent is in context.
3.2.1.1.2. What is your Reference?
On a Quote or Reference type question, if the quizmaster did not finish saying the entire reference before the quizzer jumped, then the quizmaster will prompt for the reference. For example: "What is your reference?" or "Your reference?".
3.2.1.1.3. What is your Question?
On a Reference type question, if the quizzer has given all the information in both the question and the answer and the complete question was not read, the quizmaster will prompt for the question. For example: "What is your question?" or "Question?".
3.2.1.1.4. Again
On a Quote or Finish type question, once the quizzer has quoted to the end of the required verse or verses, if the answer is not yet word-perfect, the quizmaster will say, "Again." This will occur as often as needed until the quizzer's answer is word-perfect or the answering period expires. The quizmaster is not allowed to say "Again" until the quizzer has come to the end of the verse, each time through, as this would give the quizzer information about where their mistake was made.
3.2.1.1.5. Quote is Complete
On a Situation type question, when the quizzer has completed the quotation, the quizmaster will tell the quizzer that the quote is finished. For example: "Quote is complete” or “That is the quote.”
